pythonkr_backend/pythonkr_backend/celery.py

```
@@ -0,0 +1,41 @@
+def init_beat(*args, **kwargs):
+    logfire.configure(service_name="celery-beat")  
+    logfire.instrument_celery()
+
+@app.task
+def add(x: int, y: int):
+    return x + y
+
+add.delay(42, 50)
+
+
+app.conf.beat_schedule = {  
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)